src/main/java/org/ecommerce/config/ApplicationRepositoriesConfig.java

```diff
@@ -0,0 +1,42 @@
+package org.ecommerce.config;
+
+import org.ecommerce.message.broker.MessageQueue;
+import org.ecommerce.repositories.*;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ApplicationRepositoriesConfig {
+
+    @Bean("UserRepository")
```

Comment:
Code is ok, although for code simplicity using the `@Component` annotation is better.
As experiment, please try replacing the usage of `@Bean` and put a `@Component ` annotation on each of these classes and remove the `ApplicationRepositoriesConfig` class.

Reply:
Actually I did it in a previous commit: https://github.com/algutierrez-griddynamics/e-commerce/commit/ba3f39a0bf234489152b8100b53528cce6096ad3, but the task asked me to refactor the code into an `ApplicationRepositoriesConfig`